README.md

```diff
@@ -125,6 +125,12 @@ git submodule update
 
 in the `pytype` directory.
 
+### Installing on WSL
+
+Follow the steps above, but make sure you have the correct libraries first:
+
+> sudo apt install build-essential python3-dev libpython3-dev
```

Comment:
For consistency with the rest of the file, please use a fenced code block (https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting).